testdata/project-v2/_output/config-gen.yaml

```diff
@@ -0,0 +1,392 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: (devel)
+  creationTimestamp: null
+  name: admirals.crew.testproject.org
+spec:
+  group: crew.testproject.org
+  names:
+    kind: Admiral
+    listKind: AdmiralList
+    plural: admirals
+    singular: admiral
+  scope: Cluster
+  versions:
+    - name: v1
+      schema:
+        openAPIV3Schema:
+          description: Admiral is the Schema for the admirals A
+          properties:
+            apiVersion:
+              description: 'APIVersion defines the versioned schema '
+              type: string
+            kind:
+              description: 'Kind is a string value representing the '
+              type: string
+            metadata:
+              type: object
+            spec:
+              description: AdmiralSpec defines the desired state of
+              properties:
+                foo:
+                  description: Foo is an example field of Admiral.
+                  type: string
+              type: object
+            status:
+              description: AdmiralStatus defines the observed state
+              type: object
+          type: object
+      served: true
+      storage: true
+      subresources:
+        status: {}
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: (devel)
+  creationTimestamp: null
+  name: captains.crew.testproject.org
+spec:
+  group: crew.testproject.org
+  names:
+    kind: Captain
+    listKind: CaptainList
+    plural: captains
+    singular: captain
+  scope: Namespaced
+  versions:
+    - name: v1
+      schema:
+        openAPIV3Schema:
+          description: Captain is the Schema for the captains A
+          properties:
+            apiVersion:
+              description: 'APIVersion defines the versioned schema '
+              type: string
+            kind:
+              description: 'Kind is a string value representing the '
+              type: string
+            metadata:
+              type: object
+            spec:
+              description: CaptainSpec defines the desired state of
+              properties:
+                foo:
+                  description: Foo is an example field of Captain.
+                  type: string
+              type: object
+            status:
+              description: CaptainStatus defines the observed state
+              type: object
+          type: object
+      served: true
+      storage: true
+      subresources:
+        status: {}
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: (devel)
```

Comment:
go/v2 and go/v3 does not support the same controller-gen version. 
The versions are defined in the Makefile. 

In this way, I understand that we should need to be able to configure the version that should be used as well. 
OR 
Could not the config-gen only generate the manifests and let the makefile targets generate controller-gen manifests?